src/EdFi.AnalyticsMiddleTier.Tests/Classes/SectionDim.cs

```diff
@@ -0,0 +1,23 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace EdFi.AnalyticsMiddleTier.Tests.Classes
+{
+    [SuppressMessage("ReSharper", "UnusedMember.Global")]
+    public class SectionDim
+    {
+        public int SchoolKey { get; set; } //(int, not null)
+        public string SectionKey { get; set; } //(nvarchar(75), not null)
+        public string SectionName { get; set; } //(nvarchar(75), not null)
+        public string SessionName { get; set; } //(nvarchar(75), not null)
+        public string LocalCourseCode { get; set; } //(nvarchar(75), not null)
+        public int SchoolYear { get; set; } //(int, not null)
+        public string EducationalEnvironmentTypeId { get; set; } //(nvarchar(75), not null)
```

Comment:
Hi Ben, in the DS2 version it could be string that is why I created as string.

Reply:
@NSD-Ruben do we need to be consistent between the DS2 and DS3 views to have the same columns have the same type so that it matches this definition of it? So in this case, change the DS31 and DS32 SQL to cast this column to nvarchar(20) ? I am not sure, but maybe @jleiva-gap knows ?

Reply:
I think we should replace the EducationalEnvironmentDescriptorId  by its Descriptor.Description and rename this column (EducationalEnvironmentDescriptor).

Reply:
updated